spec/index.html

```diff
@@ -225,30 +301,29 @@
           attribute boolean? expanded;
           attribute boolean? disabled;
           attribute InvalidType? invalid;
-          attribute boolean hasPopUp;
+          attribute boolean? hasPopUp;
+          attribute boolean? pressed;
 
           // Control values.
-          attribute DOMString? value;
+          attribute DOMString? valueText;
           attribute DOMString? placeholder;
-          attribute double? rangeValue;
-          attribute double? rangeMin;
-          attribute double? rangeMax;
-          attribute double? rangeStep;
+          attribute double? valueNow;
+          attribute double? valueMin;
+          attribute double? valueMax;
```

Comment:
At the F2F, we changed these to rangeValue, etc because they were more clear. Is the goal just to reduce author adoption complexity despite the odd names of some ARIA properties?